camera.js

```diff
@@ -4000,7 +4027,7 @@ app.post(['/','/:screen'],function (req,res){
                                         //new ldap login
                                         s.log({ke:req.body.key,mid:'$USER'},{type:r.lang['LDAP User is New'],msg:{info:r.lang['Creating New Account'],user:user}})
                                         req.resp.lang=r.lang
-                                        s.sqlQuery('INSERT INTO Users (ke,uid,auth,mail,pass,details) VALUES (?,?,?,?,?,?)',user.post)
+                                        s.sqlQuery('INSERT INTO Users (ke,uid,auth,mail,pass,details) VALUES (?,?,?,?,?,?)',user.post)//Why are you storing the ldap password??
```

Comment:
i'm not actually sure why, probably because i copy and pasted most of this command from somewhere else in the code that needed to save the password. I guess it isn't needed since it i need ldap to pass the success, not shinobi.

I'll remove that, thank you :D